cpp/src/aztec/honk/circuit_constructors/standard_circuit_constructor.hpp

```diff
@@ -32,6 +32,11 @@ class StandardCircuitConstructor : public CircuitConstructorBase<STANDARD_HONK_W
         // all future zero values.
         // TODO(Adrian): This should be done in a constant way, maybe by initializing the constant_variable_indices map
         zero_idx = put_constant_variable(barretenberg::fr::zero());
+        // TODO(Cody): Ensure that no polynomial is ever zero. Maybe there's a better way.
+        one_idx = put_constant_variable(barretenberg::fr::one());
+        // 1 * 1 * 1 + 1 * 1 + 1 * 1 + 1 * 1 + -4
+        // m           l       r       o        c
+        create_poly_gate({ one_idx, one_idx, one_idx, 1, 1, 1, 1, -4 });
```

Comment:
The addition of these two gates required shifting some indices by 2 in some tests where we access particular indices of the polynomial cache.